examples/resources/network_device/example_9.tf

```diff
@@ -0,0 +1,45 @@
+# Create PA-VM firewall cluster with Panorama Server Integration
```

Comment:
This won't automatically be picked up by the docs generator.  You will need to update `templates/resources/network_device.md` to add this example to the template.

To have the new attributes show up in the generated docs, you would have to either add them to the template manually or replace [these lines in the template](https://github.com/equinix/terraform-provider-equinix/blob/main/templates/resources/network_device.md.tmpl#L41-L176) with `{{ .SchemaMarkdown | trimspace }}`.  A similar change would need to be made in `templates/data-sources/network_device.md` in order to automatically generate attributes in the docs for that data source.

Once the templates are updated you will need to run `make docs` and commit the changes.

Reply:
thanks @ctreatma , looks like now these are generated rightly and Go Tests / Validate docs (pull_request)  check is passed.